docs/en/guides/common-tips.md

```diff
@@ -120,16 +119,16 @@ const $route = {
   path: '/',
   hash: '',
   params: { id: '123' },
-  query: { q: 'hello' }
+  query: { q: 'bonjour' }
 }
 
 mount(Component, {
   mocks: {
-    $route // adds the mocked $route object to the Vue instance before mounting component
+    $route // ajoute l'objet route simulé à l'instance de Vue avant de monter le composant
   }
 })
 ```
 
-## Dealing with Routing
+## Gérer le routage
 
-Since routing by definition has to do with the overall structure of the application and involves multiple components, it is best tested via integration or end-to-end tests. For individual components that rely on `vue-router` features, you can mock them using the techniques mentioned above.
+Depuis que le routage, par définition, porte sur la structure générale de l'application et implique plusieurs composants. Il est mieux testé via des tests d'intégration ou point à point (end-to-end). Pour des composants individuels qui se fie aux fonctionnalités de `vue-router`, vous pouvez les simuler en utilisant les techniques mentionnées plus haut.
```

Comment:
« end-to-end » pour rester consistant avec les autres docs.

Et je t'ai dit des conneries, c'est de « bout en bout » : https://fr.nuxtjs.org/guide/development-tools/

Vraiment désolé :(